### PR TITLE
feat: add create react app template

### LINKED ISSUE #1: Setup the Create-React-App Template
## Description 

Setup a Create React App Template. If this is the first time, you will be using React, let me share some information about it. 

Create React App (CRA) is a toolkit that is generally used for setting up React Application with ease along with a lot of features, with a unified CLI. Feel free to read about it on the official [Github Repository](https://github.com/facebook/create-react-app) and on the [Docs](https://create-react-app.dev/docs/getting-started/). This [Article](https://www.sitepoint.com/create-react-app/) can give you more depth about what you will be dealing with. 

## Tools 

- Terminal 
- NodeJS
- NPM 

## Directory Structure 

You are supposed to fork the repository and create a branch where you will push your changes. Don't push the entire App on the Root. We will follow this directory structure: 

```
- /client 
-- /src
...
-- index.js
-- registerServiceWorker.js
- .gitignore
- package-lock.json
- package.json

- /server
- /models
-- item.js
- /node.modules
``` 

The Frontend Code will be in a `client` directory while the Backend Code will be in the `server` directory. Keep all the Frontend Code on the Client Directory which you will be supposed to create. Take a look at the Boilerplate Code and replace it with a simple Heading Tag which says: `All Notes App`.

### PR DESCRIPTION
**Pull request by priyalweb**

### CODE REVIEW

**Comment by HarshCasper** [comment]
Good Work @priyalweb 🥳 

Slight Changes: 

- Please rename the commit as `feat: add create react app template`
- Please remove `.eslintcache` from the directory

File: client/src/App.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
```
Comment (HarshCasper) [inline client/src/App.js]: You may replace this with a Header tag signifying something like `Hello, All-Notes` 😄

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 1 above)
Comment (priyalweb) [reply client/src/App.js]: okay, I made the changes now 👍🏼

**Comment by HarshCasper** [comment]
Looks good to me @priyalweb 

Can you please squash all your commits before I merge this Pull Request? I have passed down the instructions on Slack if you would like to take a look.

**Review by HarshCasper — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**